docs/api/sql/Function.md

```diff
@@ -576,6 +576,46 @@ Result:
 +---------------------------------------------------------------+
 ```
 
+## ST_Force3D
+Introduction: Forces the geometry into a 3-dimensional model so that all output representations will have X, Y and Z coordinates.
+An optionally given zValue is tacked onto the geometry if the geometry is 2-dimensional. Default value of zValue is 0.0
+If the given geometry is 3-dimensional, no change is performed on it.
+If the given geometry is empty, no change is performed on it.
+
+Format: `ST_Force3D(geometry, zValue)`
+
+Since: `1.4.1`
+
+Spark SQL Example:
+
+```sql
+SELECT ST_Force3D(geometry) AS geom
+```
+
+Input: `LINESTRING(0 1, 1 2, 2 1)`
+
+Output: `LINESTRING(0 1 0, 1 2 0, 2 1 0)`
```

Comment:
Isn't the output `LINESTRING Z`?

Reply:
Yes you're right, updated the documentation with a note explaining the output